checkout-extension/shopify.extension.toml.liquid

```diff
@@ -1,7 +1,7 @@
 # Learn more about configuring your checkout UI extension:
 # https://shopify.dev/api/checkout-extensions/checkout/configuration
 
-api_version = "2023-07"
+api_version = "unstable"
```

Comment:
This can stay `2023-07` because the deployment validation will still pass - it's only the package.json that needs to change